segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLogFactory.java

```diff
@@ -189,6 +190,8 @@ private BookKeeper startBookKeeperClient() throws Exception {
             config.setEnforceMinNumRacksPerWriteQuorum(this.config.isEnforceMinNumRacksPerWriteQuorum());
             config.setMinNumRacksPerWriteQuorum(this.config.getMinNumRacksPerWriteQuorum());
             config.setProperty(CommonConfigurationKeys.NET_TOPOLOGY_SCRIPT_FILE_NAME_KEY, this.config.getNetworkTopologyFileName());
+        } else {
+            config = config.setEnsemblePlacementPolicy(DefaultEnsemblePlacementPolicy.class);
```

Comment:
Is the reason for this change related to the below WARN message observed on Segment store ?
```
2021-02-26 05:38:20,547 13644 [core-1] WARN  o.a.b.client.BookieWatcherImpl - New ensemble: [nautilus-bookie-2.nautilus-bookie-headless.nautilus-pravega.svc.cluster.local:3181, nautilus-bookie-0.nautilus-bookie-headless.nautilus-pravega.svc.cluster.local:3181, nautilus-bookie-1.nautilus-bookie-headless.nautilus-pravega.svc.cluster.local:3181] is not adhering to Placement Policy. quarantinedBookies: []
```

Reply:
Yes.